docker/Dockerfile

```diff
@@ -193,13 +192,12 @@ RUN rosdep keys --dependency-types=exec --ignore-src --from-paths src \
 FROM base AS core-devel
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 ARG ROS_DISTRO
-ARG SETUP_ARGS
```

Comment:
Why not keep `${SETUP_ARGS}` as the Dockerfile might be called outside the workflow? (to keep the Dockerfile generic)

Reply:
I hope we don't mind `SETUP_ARGS` because we can create both `no-cuda` images and `cuda` images as a multi-stage build in a single Dockerfile. If you only want to build a specific image locally, you can specify it using `docker build --target`.